### DISCUSSION TITLE
Documentation suggestion: slight modification under OAuth2 integration

### DISCUSSION THREAD

**Discussion by fercod** [pocketbase/pocketbase#3334] [Q&A] [answered]
I would remove the following sentence [from here](https://pocketbase.io/docs/authentication/#oauth2-integration), as it only applies for the _all-in-one_ solution and can cause confusion. 
In _manual code exchange_, redirect URL is handled by a custom endpoint defined by the developer.

`The default redirect URL is http://127.0.0.1:8090/api/oauth2-redirect.`

**Comment by ganigeorgiev** [comment, marked as answer]
I'll consider updating it sometime later today (_maybe with a tooltip next to "...  and register a redirect URL" that mentions both options_).
